tests/installation/grub_test.pm

```diff
@@ -81,6 +81,8 @@ sub bug_workaround_bsc1005313 {
 sub run {
     my ($self) = @_;
     my $timeout = get_var('GRUB_TIMEOUT', 90);
+    # on IPMI keys are sometimes lost in the bootloader
+    my $retry = check_var('BACKEND','ipmi') ? 1 : undef); 
```

Comment:
extra `)`
You might not even need `$retry` var since it's used only once - but whatever is more readable for you.